tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -244,6 +255,24 @@ def set_route53_with_eks_ingress():
     time.sleep(60)
 
 
+def set_route53_with_aks_ingress():
```

Comment:
I am confused here: Why do you use AWS's route53 for an AKS cluster?

Reply:
DNS is platform agnostic, and we have the logic to manage route53 already. 
Second reason the DNS in Azure is global and I don't want to touch it if possible.

Reply:
In this method it calls `AmazonWebServices().upsert_route_53_record_cname` which is from the AWS module. How does it work with AKS?